src/TTGamesExplorerRebirthUI/Helper.cs

```diff
@@ -28,44 +28,59 @@ public static string FormatSize(ulong bytesSize)
             return string.Format("{0:n1}{1}", number, sizeSuffixes[counter]);
         }
 
+        static readonly private Bitmap bitmap_page_white_music = new Bitmap(Properties.Resources.page_white_music);
```

Comment:
If it’s private it should be 
```suggestion
        static readonly private Bitmap _bitmapPageWhiteMusic = new Bitmap(Properties.Resources.page_white_music);
```
all same for all other fields